### PR TITLE
update postivity commondata

### PR DESCRIPTION
**Pull request by RoyStegeman**
Since we cut some points for nfonll we need to remove them from the commondata as well. Of course, since this is shared for all theories that's going to cause clashes. Not sure if there is a good solution for this?

### CODE REVIEW

**Comment by RoyStegeman** [comment]
That would be a good solution, thanks.

**Comment by scarlehoff** [comment]
Alternatively (if you are in a rush for the tags) you can also create two new positivity datasets _SHORT or something and update the yamldb in the relevant theories.

**Comment by RoyStegeman** [comment]
I'd rather wait. It's not me who's in a hurry anyway, but I'll probably be the one who's asked about it on Friday ;)

File: nnpdfcpp/data/commondata/DATA_POSFLL.dat [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
+  11	DIS_FLL	1.800000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+  12	DIS_FLL	2.600000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+  13	DIS_FLL	3.400000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+  14	DIS_FLL	4.200000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+  15	DIS_FLL	5.000000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+  16	DIS_FLL	5.800000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+  17	DIS_FLL	6.600000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+  18	DIS_FLL	7.400000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+  19	DIS_FLL	8.200000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+  20	DIS_FLL	9.000000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
```
Comment (RoyStegeman) [inline nnpdfcpp/data/commondata/DATA_POSFLL.dat:20]:
```suggestion
POSFLL	0	19
   1	DIS_FLL	1.940766723678e-06	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
   2	DIS_FLL	7.533150951473e-06	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
   3	DIS_FLL	2.924017738213e-05	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
   4	DIS_FLL	1.134967265154e-04	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
   5	DIS_FLL	4.405413401349e-04	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
   6	DIS_FLL	1.709975946677e-03	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
   7	DIS_FLL	6.637328831201e-03	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
   8	DIS_FLL	2.576301385941e-02	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
   9	DIS_FLL	1.000000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
  10	DIS_FLL	1.800000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
  11	DIS_FLL	2.600000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
  12	DIS_FLL	3.400000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
  13	DIS_FLL	4.200000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
  14	DIS_FLL	5.000000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
  15	DIS_FLL	5.800000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
  16	DIS_FLL	6.600000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
  17	DIS_FLL	7.400000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
  18	DIS_FLL	8.200000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
  19	DIS_FLL	9.000000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
```

File: nnpdfcpp/data/commondata/DATA_POSFLL.dat [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
+POSFLL	0	19
+   1	DIS_FLL	1.940766723678e-06	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+   2	DIS_FLL	7.533150951473e-06	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+   3	DIS_FLL	2.924017738213e-05	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+   4	DIS_FLL	1.134967265154e-04	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+   5	DIS_FLL	4.405413401349e-04	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+   6	DIS_FLL	1.709975946677e-03	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+   7	DIS_FLL	6.637328831201e-03	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+   8	DIS_FLL	2.576301385941e-02	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
+   9	DIS_FLL	1.000000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
```
Comment (RoyStegeman) [inline nnpdfcpp/data/commondata/DATA_POSFLL.dat:10]:
```suggestion
  9	DIS_FLL	1.000000000000e-01	5.000000000000e+00	0.000000000000e+00	0.000000000000e+00	0.000000000000e+00
```

**Comment by scarlehoff** [comment]
Having had a look, I think it would be much better to have these as separate positivity datasets.

Since the theories are not yet updated with the new fktables, they can just be added with a `_whatever` suffix and the same for the yamldb (cc @andreab1997 @niclaurenti). Also because I'm foreseeing situations in which the same fit is to be run several times with and without these points...

**Comment by andreab1997** [comment]
> Having had a look, I think it would be much better to have these as separate positivity datasets.
> 
> Since the theories are not yet updated with the new fktables, they can just be added with a `_whatever` suffix and the same for the yamldb (cc @andreab1997 @niclaurenti). Also because I'm foreseeing situations in which the same fit is to be run several times with and without these points...

Unfortunately now the theories are updated...

**Comment by scarlehoff** [comment]
> Unfortunately now the theories are updated...

Time to update again :P 
Luckily is only a change of name in the yamldb (no need to have both fktables if they are not to be used).

**Comment by andreab1997** [comment]
> > Unfortunately now the theories are updated...
> 
> Time to update again :P Luckily it's only a change of name in the yamldb (no need to have both fktables if they are not to be used).

Ok we only need to change the same of the yamldb, right?

**Comment by andreab1997** [comment]
The theories have been updated. If you notice some error please let me know and I'll fix it

**Comment by RoyStegeman** [comment]
@andreab1997 this should be good now. 

I didn't add them to buildmaster.

**Review by andreab1997 — APPROVED** [review]
Looks good to me

**Comment by scarlehoff** [comment]
Before merging (and adding them a buildmaster, which is still necessary) please make sure that they 1) install correctly 2) and work.

**Pull request merged**